ross/materials.py

```diff
@@ -47,21 +47,28 @@ class Material:
 
     """
 
-    def __init__(self, name, rho, **kwargs):
-
-        assert name is not None, "Name not provided"
-        assert type(name) is str, "Name must be a string"
-        assert " " not in name, "Spaces are not allowed in Material name"
-        assert (
-            sum([1 if i in ["E", "G_s", "Poisson"] else 0 for i in kwargs]) > 1
-        ), "At least 2 arguments from E, G_s and Poisson should be provided"
-
+    def __init__(
+        self, name, rho, E=None, G_s=None, Poisson=None, color="#525252", **kwargs
+    ):
+
+        self.name = str(name)
+        if " " in name:
+            raise ValueError("Spaces are not allowed in Material name")
+
+        given_args = []
+        for arg in ["E", "G_s", "Poisson"]:
+            if locals()[arg] is not None:
+                given_args.append(arg)
+        if len(given_args) != 2:
+            raise ValueError(
+                "At least 2 arguments from E, G_s" "and Poisson should be provided "
+            )
```

Comment:
This error message could lead for a wrong conclusion. When using "at least" the user could think we could input two or all the three parameters.
Maybe we could say something more restrictive like:
 ```
raise ValueError(
    "Only 2 arguments from E, G_s and Poisson should be provided."
)
```

Reply:
You are right. I will change this and update the PR.